core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -4571,7 +4571,7 @@ private void checkEmptyJoin(RelOptFixture f) {
       final RelDataType type = rexBuilder.getTypeFactory().createSqlType(SqlTypeName.BIGINT);
 
       RelNode left = b
-          .values(new String[]{"x", "y"}, 1, 2).build();
+          .values(new String[]{"x", "y"}, 1, 2, 2, 1).build();
```

Comment:
I added a second row to prevent `RexSimplify` from seeing the columns as constants and reducing this to `true`.